src/qt/overviewpage.cpp

```diff
@@ -132,7 +132,8 @@ OverviewPage::OverviewPage(const PlatformStyle *platformStyle, QWidget *parent)
     currentWatchOnlyBalance(-1),
     currentWatchUnconfBalance(-1),
     currentWatchImmatureBalance(-1),
-    txdelegate(new TxViewDelegate(platformStyle, this))
+    txdelegate(new TxViewDelegate(platformStyle, this)),
+    timer(nullptr)
```

Comment:
`timer` should be initialised first in the list.
gcc 7.1 gives this warning:
```
In file included from ../../src/qt/overviewpage.cpp:6:0:
../../src/qt/overviewpage.h: In constructor ‘OverviewPage::OverviewPage(const PlatformStyle*, QWidget*)’:
../../src/qt/overviewpage.h:64:21: warning: ‘OverviewPage::txdelegate’ will be initialized after [-Wreorder]
     TxViewDelegate *txdelegate;
                     ^~~~~~~~~~
../../src/qt/overviewpage.h:50:13: warning:   ‘QTimer* OverviewPage::timer’ [-Wreorder]
     QTimer *timer;
             ^~~~~
../../src/qt/overviewpage.cpp:124:1: warning:   when initialized here [-Wreorder]
 OverviewPage::OverviewPage(const PlatformStyle *platformStyle, QWidget *parent) :
 ^~~~~~~~~~~~
```

Reply:
Yep, noticed this as well. However, a) this is not critical imo and b) should be fixed in `develop`, not here.